packages/color-area/src/ColorArea.ts

```diff
@@ -0,0 +1,386 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    html,
+    SpectrumElement,
+    CSSResultArray,
+    TemplateResult,
+    property,
+    PropertyValues,
+    query,
+    streamingListener,
+} from '@spectrum-web-components/base';
+import { ColorHandle } from '@spectrum-web-components/color-handle';
+import '@spectrum-web-components/color-handle/sp-color-handle.js';
+import { HSL, HSLA, HSV, HSVA, RGB, RGBA, TinyColor } from '@ctrl/tinycolor';
+
+import styles from './color-area.css.js';
+
+/**
+ * @element sp-color-area
+ */
+export class ColorArea extends SpectrumElement {
+    public static get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public disabled = false;
+
+    @property({ type: Boolean, reflect: true })
+    public focused = false;
+
+    @query('.handle')
+    private handle!: ColorHandle;
+
+    @property({ type: Number })
+    public get hue(): number {
+        return this._hue;
+    }
+
+    public set hue(value: number) {
+        const hue = Math.min(360, Math.max(0, value));
+        if (hue === this.hue) {
+            return;
+        }
+        const oldValue = this.hue;
+        const { s, v } = this._color.toHsv();
+        this._color = new TinyColor({ h: hue, s, v });
+        this._hue = hue;
+        this.requestUpdate('hue', oldValue);
+    }
+
+    private _hue = 0;
+
+    @property({ type: String })
+    public get color():
```

Comment:
Did you mean the `hue` property? That's where I see the discrepancy between `ColorArea` and the other elements.

Reply:
For `sp-color-area` the "value" is `color`, I _think_. The duplication here would allow `sp-color-area` to act like an `input` wherein you are able to say `document.querySelector('input').value`. With the goal of reducing the need for specific domain API knowledge. However, if that feels weird, we can revisit this later as it is expanded API, not required, IMO.

Reply:
I see! I'll check that out.